examples/module1/lesson1/solver/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Great job with introducing this presentational component // Greetings from recording ;)